src/components/pages/areas-manage/AreasManage.js

```diff
@@ -135,7 +135,7 @@ class AreasManage extends React.Component {
                     />
                 </div>
               </div>
-              <Button className="c-button">Save</Button>
+              <Button className="c-button" disabled={this.props.saving}>Save</Button>
```

Comment:
It will be nice to disable the button also when the name input is empty or undefined (this.form.name === '') and when the editing mode is on so we don't have two save buttons available at the same time